### PR TITLE
Updated Docker configuration for Pluto 0.12

### PR DESCRIPTION
**Pull request by lungben**
The changes in the Pluto config management required some modifications in the startup files.

```
security_options = Pluto.Configuration.SecurityOptions(; require_secret_for_access=false)
```
I have not switched off `require_secret_for_open_links` for now, but I am not sure if this is the best configuration for Docker.
What do you think?

### CODE REVIEW

**Comment by fonsp** [comment]
Yep that's the right combination of configs!

**Comment by lungben** [comment]
The standard (non-precompiled) Docker image is updated and seems to work (I did only a spot check however).
Building of the precompiled Docker image failed, however, with an error message in the warmup.jl script.

**Comment by fonsp** [comment]
Can you share the error message?

**Comment by lungben** [comment]
Thanks, I think I got it now - the container is building locally on my machine

**Review by shashankp — APPROVED** [review]

File: docker/precompiled/prestartup.jl [hunk 1]
```diff
@@ -1,2 +1,3 @@
 using Pkg
 Pkg.add(["Pluto", "PlutoUI", "Plots"]) # "Images", "ImageIO", "ImageMagick"])
```
[the remark this answers is not in the document]
Comment (lungben) [reply docker/precompiled/prestartup.jl]: Thanks, is removed.

**Review by lungben — COMMENTED** [review]
@shashankp, @fonsp thanks for your review, merging now.
The Docker images have been updated (still manually) for Pluto 0.12.1, I'll trigger building for 0.12.2 now.

**Pull request merged**